topics/assembly/tutorials/ecoli_comparison/tools.yaml

```diff
@@ -0,0 +1,9 @@
+---
+api_key: admin
+galaxy_instance: http://localhost:8080
+install_resolver_dependencies: true
+install_tool_dependencies: false
+tools:
```

Comment:
This file needs to be filed. The tools are listed in the workflows with revision number.

Reply:
so I really need and catalog all tools? Man, no one will write tutorials if this is a requirement.

Reply:
Sorry for my public venting ...

Reply:
If you have a workflow, you can generate the `tools.yaml` file: see http://galaxyproject.github.io/training-material/topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html#filling-the-toolsyaml